config/nginx.conf

```diff
@@ -17,6 +17,7 @@ http {
     error_log /dev/stderr error;
 
     log_format main_spectre '{"time_iso8601": "$time_iso8601", "remote_addr": "$remote_addr", "connection": "$connection", "connection_requests": "$connection_requests", "http_user_agent": "$http_user_agent", "status": "$status", "request_length": "$request_length", "bytes_sent": "$bytes_sent", "request_time": "$request_time", "request_uri": "$request_uri", "unix_time": "$msec", "x_smartstack_source": "$http_x_smartstack_source", "x_smartstack_destination": "$http_x_smartstack_destination", "spectre_cache_status": "$sent_http_spectre_cache_status"}';
+    # Update start.sh script if you change next line
     access_log /dev/stdout main_spectre;
```

Comment:
Why not just replace `/dev/stdout` with a known placeholder that you can then replace rather than match on the entire line?

Reply:
That's what I would do if you have a strong opinion about the previous comment